eng/packages/http-client-csharp/eng/scripts/Generation.psm1

```diff
@@ -57,23 +57,75 @@ function Get-TspCommand {
     return $command
 }
 
+function Get-Mgmt-TspCommand {
+    param (
+        [string]$specFile,
+        [string]$generationDir,
+        [bool]$generateStub = $false,
+        [string]$namespaceOverride = $null,
+        [string]$apiVersion = $null,
+        [bool]$forceNewProject = $false
+    )
+    $command = "npx tsp compile $specFile"
+    $command += " --trace @azure-typespec/http-client-csharp-mgmt"
+    $command += " --emit @azure-typespec/http-client-csharp-mgmt"
+    $configFile = Join-Path $generationDir "tspconfig.yaml"
+    if (Test-Path $configFile) {
+        $command += " --config=$configFile"
+    }
+    $command += " --option @azure-typespec/http-client-csharp-mgmt.emitter-output-dir=$generationDir"
+    $command += " --option @azure-typespec/http-client-csharp-mgmt.save-inputs=true"
+    if ($generateStub) {
+        $command += " --option @azure-typespec/http-client-csharp-mgmt.plugin-name=AzureStubPlugin"
+    }
+
+    if ($namespaceOverride) {
+        $command += " --option @azure-typespec/http-client-csharp-mgmt.namespace=$namespaceOverride"
```

Comment:
Is this option actually supported yet? I think we support the package-name in MTG, not namespace. Namespace support would need to be added in the azure emitter.

Reply:
I guess @m-nash copied this from MGC in the first place. 
If this is not supported in azure yet, I can remove the corresponding logic of namespace override in this script.

Reply:
The support was removed from MTG as part of the namespace is namespace feature. I don't think it is supported in Azure generator yet.

Reply:
This namespaceoverride has been removed from the script.